src/panels/config/integrations/integration-panels/zwave_js/zwave_js-custom-param.ts

```diff
@@ -0,0 +1,219 @@
+                class="error-icon"
+                slot="item-icon"
+              ></ha-svg-icon
+              ><em>${this._error}</em>`
+          : nothing}
+      </div>
+    `;
+  }
+
+  private _customParamNumberChanged(ev: Event) {
+    this._customParamNumber =
+      Number((ev.target as HTMLInputElement).value) || undefined;
+  }
+
+  private _customValueSizeChanged(ev: Event) {
+    this._valueSize = Number((ev.target as HTMLSelectElement).value) || 1;
+  }
+
+  private _customValueChanged(ev: Event) {
+    this._value = Number((ev.target as HTMLInputElement).value) || undefined;
+  }
+
+  private _customValueFormatChanged(ev: Event) {
+    this._valueFormat = Number((ev.target as HTMLSelectElement).value) || 0;
+  }
+
+  private async _getCustomConfigValue() {
+    if (this._customParamNumber === undefined) {
+      this._error = this.hass.localize(
+        "ui.panel.config.zwave_js.node_config.error_required",
+        {
+          entity: this.hass.localize(
+            "ui.panel.config.zwave_js.node_config.parameter"
+          ),
+        }
+      );
+      return;
+    }
+    this._error = "";
+    try {
+      const value = await getZwaveNodeRawConfigParameter(
+        this.hass,
+        this.deviceId,
+        this._customParamNumber
+      );
+      this._value = value;
+    } catch (err: any) {
+      this._error = err?.message || "Unknown error";
+    }
+  }
+
+  private async _setCustomConfigValue() {
+    if (this._customParamNumber === undefined) {
+      this._error = this.hass.localize(
+        "ui.panel.config.zwave_js.node_config.error_required",
+        {
+          entity: this.hass.localize(
+            "ui.panel.config.zwave_js.node_config.parameter"
+          ),
+        }
+      );
+      return;
+    }
+    if (this._value === undefined) {
+      this._error = this.hass.localize(
+        "ui.panel.config.zwave_js.node_config.error_required",
+        {
+          entity: this.hass.localize(
+            "ui.panel.config.zwave_js.node_config.value"
+          ),
+        }
+      );
+      return;
+    }
+    this._error = "";
+    try {
+      await setZwaveNodeRawConfigParameter(
+        this.hass,
+        this.deviceId,
+        this._customParamNumber,
+        this._value,
+        this._valueSize,
+        this._valueFormat
+      );
+    } catch (err: any) {
+      this._error = err?.message || "Unknown error";
+    }
+  }
+
+  static get styles(): CSSResultGroup {
+    return css`
+      .custom-config-form {
+        display: flex;
+        flex-wrap: wrap;
+        gap: 16px;
+        margin-bottom: 16px;
+      }
+
+      ha-textfield,
+      ha-select {
+        flex-grow: 1;
+        flex-basis: calc(50% - 8px);
+        min-width: 120px;
+      }
+
+      @media (min-width: 681px) {
+        .custom-config-form {
+          flex-wrap: nowrap;
+        }
+
+        ha-textfield,
+        ha-select {
+          flex-basis: 0;
+        }
+      }
+
```

Comment:
```suggestion
.custom-config-buttons {
  display: flex;
  justify-content: flex-end;
}
```

As far as I understood our design guidelines action buttons should be placed on the bottom right.